data_submission/src/encipher/avail_client.rs

```diff
@@ -0,0 +1,52 @@
+use avail_rust::{error::ClientError, Block, Filter, H256, SDK};
+use std::str::FromStr;
+
```

Comment:
we already have a crate named `avail` that does this. Omit this file and reuse the code.